specification/eventgrid/data-plane/readme.md

```diff
@@ -333,13 +333,14 @@ input-file:
   - $(this-folder)/Microsoft.ResourceNotifications/stable/2018-01-01/common.json
   - $(this-folder)/Microsoft.ResourceNotifications/stable/2018-01-01/HealthResources.json
   - $(this-folder)/Microsoft.ResourceNotifications/stable/2018-01-01/Resources.json
+  - $(this-folder)/Microsoft.AVS/preview/2023-09-01-preview/PrivateCloud.json
 ```
 
 If there are files that should not be in the `all-api-versions` set, 
 uncomment the  `exclude-file` section below and add the file paths.
 
 ``` yaml $(tag) == 'all-api-versions'
-#exclude-file: 
-#  - $(this-folder)/Microsoft.Example/stable/2010-01-01/somefile.json
+exclude-file: 
+ - $(this-folder)/Microsoft.EventGrid/preview/2023-06-01-preview/examples
```

Comment:
Why was this changed?

Reply:
I tried following the instructions to avoid triggering Avocado failure, I'll undo the change.